ui/app/src/components/PreviewDialog/utils.ts

```diff
@@ -28,6 +28,7 @@ export interface PreviewDialogBaseProps {
   content?: string;
   mimeType?: string;
   backgroundModeIndex?: number;
+  hideEdit?: boolean;
```

Comment:
Default to true on the component.
```suggestion
  showEdit?: boolean;
```